DiscordBot.DataAccess/EventsSheetsService.cs

```diff
@@ -548,6 +557,13 @@ IEnumerable<EventResponse> responseColumns
         {
             var index = responseColumns.ToList().FindIndex(responseOption => responseOption.Emoji == responseEmoji);
 
+            if (index == -1)
+            {
+                throw new ResponseNotFoundException(
+                    $"Response {responseEmoji} is not recognised for event {eventKey}"
+                );
+            }
```

Comment:
Does this check really need to be duplicated in both add response methods? It looks like the logic is really the same for both so could be done for both of them in the calling method? What do you think? 🙂

Reply:
Yeah I agree! Changed in 32ae737 🙂